src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -348,7 +357,7 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
   return withDisplayName('wrapWorkspace', Wrapper)
 }
 
-export const WorkspaceMenuTrigger = ({ children, canShare, isOwner, setCloningWorkspace, setSharingWorkspace, setDeletingWorkspace }) => h(
+export const WorkspaceMenuTrigger = ({ children, canShare, isLocked, namespace, name, isOwner, setCloningWorkspace, setSharingWorkspace, setDeletingWorkspace, setTogglingWorkspaceLock }) => h(
```

Comment:
`namespace` and `name` are not used.

Also, this looks like a [duplication](https://github.com/DataBiosphere/terra-ui/blob/f0f6fd43ef964d7fed9851af433a6f1ddb01efa2/src/pages/workspaces/List.js#L50) of the workspace menu in the workspaces list.

While it may be out of scope for this PR, could you add a PR to your board to refactor these menus into a shareable component?

Reply:
Unused values removed in 54bcb3b2f4cdbce8d945a34571efc72802a61cfb

I'll add a ticket to our backlog for dealing with the code duplication